tests/unit/remote/test_s3.py

```diff
@@ -6,6 +6,8 @@
 bucket_name = "bucket-name"
 prefix = "some/prefix"
 url = f"s3://{bucket_name}/{prefix}"
+key_id = "key_id"
+key_secret = "key_secret"
```

Comment:
a bit worried of some nasty bug like that :smiley: 
```suggestion
key_id = "key-id"
key_secret = "key-secret"
```